diesel/src/result.rs

```diff
@@ -66,7 +66,13 @@ pub enum Error {
     ///
     /// An example of when this error would be returned is if a rollback has
     /// already be called on the current transaction.
-    RollbackError(Box<Error>),
+    RollbackError {
+        /// The error that caused the rollback failure
+        rollback_error: Box<Error>,
```

Comment:
I would like to improve the name of this field, so that it is clearer that this is the error that **caused** the rollback instead of the error that was returned by an failed rollback. Maybe `rollback_cause` or something like that